### PR TITLE
exchange.assignDefaultDepositWithdrawFees created

### PR DESCRIPTION
**Pull request by samgermain**
- Takes a depositWithdrawFee structure and assigns the default values for withdraw and deposit
- determines what the default network is by 
    - checking if there is only one network
    - checking if the network is equal to the currency code (or ERC20 for ETH, TRC20 for TRX, ...)

### CODE REVIEW

File: js/base/Exchange.js [hunk 1]
```diff
@@ -2972,4 +2972,21 @@ module.exports = class Exchange {
...
+        /**
+         * @ignore
+         * @method
+         * @description Takes a depositWithdrawFee structure and assigns the default values for withdraw and deposit
+         * @param {object} fee A deposit withdraw fee structure
+         * @returns {object} A deposit withdraw fee structure
+         */
+        const networkKeys = Object.keys (fee['networks']);
+        const numNetworks = networkKeys.length;
+        if (numNetworks === 1) {
```
Comment (carlosmiei) [inline js/base/Exchange.js:2986]: @samgermain we should have a default regardless of the number of networks available

Comment (samgermain) [reply js/base/Exchange.js:2986]: > @samgermain we should have a default regardless of the number of networks available, right?

if the exchange doesn't tell us what the default is, then how can we determine it?

Comment (carlosmiei) [reply js/base/Exchange.js:2986]: @samgermain the main tokens with their own blockchain that could be the default, if the token is a erc20 token ERC20 would be the default and so on and so forth

Comment (samgermain) [reply js/base/Exchange.js:2991]: ok, I updated it and tested it out on the whitebit depositWithdrawFees PR and it looks like it's working 

```
2022-12-01T04:30:31.041Z
Node.js: v18.4.0
CCXT v2.2.40
whitebit.fetchDepositWithdrawFees ()
2022-12-01T04:30:33.881Z iteration 0 passed in 1198 ms

{
  '1INCH': {
    info: {
      '1INCH': {
        is_depositable: true,
        is_withdrawal: true,
        ticker: '1INCH',
        name: '1inch',
        providers: [],
        withdraw: {
          max_amount: '0',
          min_amount: '25',
          fixed: '3.14',
          flex: null
        },
        deposit: { max_amount: '0', min_amount: '15', fixed: null, flex: null }
      }
    },
    withdraw: { fee: 3.14, percentage: false },
    deposit: { fee: undefined, percentage: undefined },
    networks: {}
  },
  ...
  ETH: {
    info: {
      'ETH (ARBITRUM)': {
        is_depositable: true,
        is_withdrawal: true,
        ticker: 'ETH',
        name: 'Ethereum',
        providers: [],
        withdraw: {
          max_amount: '0',
          min_amount: '0.006',
          fixed: '0.0007',
          flex: null
        },
        deposit: {
          max_amount: '0',
          min_amount: '0.0035',
          fixed: null,
          flex: null
        }
      },
      'ETH (BEP20)': {
        is_depositable: true,
        is_withdrawal: true,
        ticker: 'ETH',
        name: 'Ethereum',
        providers: [],
        withdraw: {
          max_amount: '0',
          min_amount: '0.006',
          fixed: '0.000219',
          flex: null
        },
        deposit: {
          max_amount: '0',
          min_amount: '0.0035',
          fixed: null,
          flex: null
        }
      },
      'ETH (ERC20)': {
        is_depositable: true,
        is_withdrawal: true,
        ticker: 'ETH',
        name: 'Ethereum',
        providers: [],
        withdraw: {
          max_amount: '0',
          min_amount: '0.01',
          fixed: '0.0008964',
          flex: null
        },
        deposit: {
          max_amount: '0',
          min_amount: '0.0065',
          fixed: null,
          flex: null
        }
      },
      'ETH (OPTIMISM)': {
        is_depositable: true,
        is_withdrawal: true,
        ticker: 'ETH',
        name: 'Ethereum',
        providers: [],
        withdraw: {
          max_amount: '0',
          min_amount: '0.006',
          fixed: '0.0007',
          flex: null
        },
        deposit: {
          max_amount: '0',
          min_amount: '0.0035',
          fixed: null,
          flex: null
        }
      }
    },
    withdraw: { fee: 0.0008964, percentage: false },
    deposit: { fee: undefined, percentage: undefined },
    networks: {
      ARBITRUM: {
        withdraw: { fee: 0.0007, percentage: false },
        deposit: { fee: undefined, percentage: undefined }
      },
      BEP20: {
        withdraw: { fee: 0.000219, percentage: false },
        deposit: { fee: undefined, percentage: undefined }
      },
      ERC20: {
        withdraw: { fee: 0.0008964, percentage: false },
        deposit: { fee: undefined, percentage: undefined }
      },
      OPTIMISM: {
        withdraw: { fee: 0.0007, percentage: false },
        deposit: { fee: undefined, percentage: undefined }
      }
    }
  },
  ...
  ZRX: {
    info: {
      ZRX: {
        is_depositable: true,
        is_withdrawal: true,
        ticker: 'ZRX',
        name: '0x',
        providers: [],
        withdraw: {
          max_amount: '0',
          min_amount: '55',
          fixed: '8.55',
          flex: null
        },
        deposit: { max_amount: '0', min_amount: '45', fixed: null, flex: null }
      }
    },
    withdraw: { fee: 8.55, percentage: false },
    deposit: { fee: undefined, percentage: undefined },
    networks: {}
  }
}
2022-12-01T04:30:33.881Z iteration 1 passed in 1198 ms
```

Comment (carlosmiei) [reply js/base/Exchange.js:2991]: @samgermain we should be using ETH as the main ETH chain, `networkCodeToId` and `networkIdToCode` already take that into consideration

Comment (carlosmiei) [reply js/base/Exchange.js:2991]: @samgermain can u check this comment?

Comment (samgermain) [reply js/base/Exchange.js:2991]: I'll update it

Comment (carlosmiei) [reply js/base/Exchange.js:2991]: @samgermain when u can, pls update it 🙂

**Comment by carlosmiei** [comment]
LGTM ✅

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**